straxen/contexts.py

```diff
@@ -19,12 +19,15 @@
         straxen.PulseProcessing,
         straxen.Peaklets,
         straxen.PeakletClassification,
+        straxen.PeakletSOMClass,
```

Comment:
These three plugins registered to the default nT context are additional plugins. They will allow us to keep the already processed data as is and provide the additional SOM fields at peak and event level as additional data-types. Thus, after processing on midway we have to use OSG to process `peak_som_classification` and `event_som_classification`.